auth/src/main/java/org/aerogear/auth/impl/BrowserAuthenticatorImpl.java

```diff
@@ -0,0 +1,25 @@
+package org.aerogear.auth.impl;
+
+import org.aerogear.auth.AuthServiceConfig;
+import org.aerogear.auth.credentials.ICredential;
+
+import java.security.Principal;
+
+/**
+ * Authenticates the user by opening a browser window asking for credentials.
+ */
+public class BrowserAuthenticatorImpl extends OICDTokenAuthenticatorImpl {
```

Comment:
Given that the implementation is mainly based on OIDC protocol, I think it's better to name it `OIDCAuthenticatorImpl`?